src/main/java/common/itemBlocks/IB_TFFTStorageField.java

```diff
@@ -44,6 +44,12 @@ public void addInformation(ItemStack stack, EntityPlayer player, List lines, boo
                             + NumberFormat.getNumberInstance().format(GTMTE_TFFT.Field.VALUES[meta - 1].getCapacity())
                             + EnumChatFormatting.GRAY
                             + " L");
+            lines.add(
+                    "Per Fluid Capacity: " + EnumChatFormatting.BLUE
+                            + NumberFormat.getNumberInstance()
+                                    .format(GTMTE_TFFT.Field.VALUES[meta - 1].getCapacity() / 25)
```

Comment:
Can you make 25 a constant variable with a name? Would like for it to not be a "random number" when someone comes here and sees it with no context. the variable could be named UNIQUE_FLUID_PER_CELL or something like that

Reply:
Good call.